tesseract_kinematics/test/kinematics_core_unit.cpp

```diff
@@ -73,6 +73,13 @@ TEST(TesseractKinematicsUnit, UtilsHarmonizeUnit)  // NOLINT
   tesseract_kinematics::harmonizeTowardZero(q.data(), 2);
   EXPECT_NEAR(q[0], M_PI_4, 1e-6);
   EXPECT_NEAR(q[1], -M_PI_4, 1e-6);
+
+  q[0] = M_PI_4;
+  q[1] = -M_PI_4;
+
+  tesseract_kinematics::harmonizeTowardZero(q.data(), 2);
+  EXPECT_NEAR(q[0], M_PI_4, 1e-6);
+  EXPECT_NEAR(q[1], -M_PI_4, 1e-6);
```

Comment:
It's probably worth adding a test case for a number in the range [pi, 2 * pi] (like 5 * pi / 4) to verify correct behavior